pycyphal/transport/can/media/socketcan/_socketcan.py

```diff
@@ -123,12 +123,11 @@ def start(self, handler: Media.ReceivedFramesHandler, no_automatic_retransmissio
     def configure_acceptance_filters(self, configuration: typing.Sequence[FilterConfiguration]) -> None:
         if self._closed:
             raise pycyphal.transport.ResourceClosedError(repr(self))
-        _logger.info(
-            "%s FIXME: acceptance filter configuration is not yet implemented; please submit patches! "
-            "Requested configuration: %s",
-            self,
-            ", ".join(map(str, configuration)),
-        )
+
+        try:
+            self._sock.setsockopt(socket.SOL_CAN_RAW, socket.CAN_RAW_FILTER, _pack_filters(configuration))
```

Comment:
```suggestion
            self._sock.setsockopt(socket.SOL_CAN_RAW, socket.CAN_RAW_FILTER, _pack_filters(configuration))  # type: ignore
```

This is to fix MyPy on Windows